timelock-server/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM frolvlad/alpine-oraclejdk8:slim
+MAINTAINER AtlasDB Team
+
+RUN apk update && apk add libstdc++ bash wget
+
+# Prefer ADD to COPY because it does the tar handling
+ADD build/docker/timelock-server-snapshot.tgz /timelock-server
+
+# Remove possible version prefix
+RUN for file in timelock-server/*; do mv "$file" "timelock-server/timelock-server-snapshot"; done
```

Comment:
In sync with removing the version from the directories/files, but do we need a loop here? Can we get away with just one clever `mv` command?